govc/library/deploy.go

```diff
@@ -61,6 +63,10 @@ func (cmd *deploy) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.OptionsFlag.Register(ctx, f)
 
 	f.StringVar(&cmd.profile, "profile", "", "Storage profile")
+
+	if os.Getenv("VM_CLASS_AS_CONFIG") == "true" { // FSS
```

Comment:
Question: why do we conditionally enable (define) this flag based on this env var? Is this a pattern we already do? Would it harm to always define it, since it's `""` by default?

Also: what does the comment `// FFS` stand for?

Reply:
Feature State Switch. We use this pattern in vCenter/WCP/VM Service/Guest Cluster Service, but this is the first time in govc it'd be used for a specific flag/field.  govc does have a way to disable unreleased commands by default (see below). We'd used that for library.{checkin,checkout} for example.  This feature will be released with 8.0, but is still subject to change and could still be disabled by default?  Just wanted to add the flag for testing within govmomi itself and folks working on the feature.

https://github.com/vmware/govmomi/blob/18cdce89aae64488c0f3c76d113c3ff7c076cbb0/govc/cli/register.go#L25-L27

Reply:
On second thought, I suppose we could just use the same `GOVC_SHOW_UNRELEASED` env var here instead adding a feature specific env var.

Reply:
Pushed a change that does the above, can you review again @sreyasn @embano1 ?